swagger2locustio/__main__.py

```diff
@@ -101,6 +104,107 @@ def main():
+    # log_result(args.results_path)
+
+
+def log_diff(start, end, results_path):
+    """Function: log difference"""
+
+    for key, items in start.items():
+        start_key = set(items)
+        end_key = set(end[key])
+        result = {
+            "created": [],
+            "unchanged": [],
+            "updated": [],
+            "deleted": [],
+        }
+
+        # CREATED / DELETED
+        # cutting of multiline to avoid issues with code that was changed
+        start_key_names = {x[: x.find("\n")] if x.find("\n") != -1 else x for x in start_key}
+        end_key_names = {x[: x.find("\n")] if x.find("\n") != -1 else x for x in end_key}
+        result["created"] = list(end_key_names - start_key_names)
+        result["deleted"] = list(start_key_names - end_key_names)
+
+        # UNCHANGED / UPDATED
+        for each_start in start_key:
+            for each_end in end_key:
+                if each_start == each_end:
+                    # folder entries do not include \n, so last letter is being cut off
+                    result["unchanged"].append(
+                        each_start[: each_start.find("\n")] if each_start.find("\n") != -1 else each_start
+                    )
+                elif each_start[: each_start.find("\n")] == each_end[: each_end.find("\n")]:
+                    result["updated"].append(each_start[: each_start.find("\n")])
+                # else is not used as we compare two lists which includes a lot of false entries
+
+        for result_key in result:
+            result[result_key].sort()
+            result_len = len(result[result_key])
+            if result_len != 0 and (key != "folders" or result_key != "updated"):
+                logging.info("%s %s: %d", key.upper(), result_key, result_len)
+                if result_key != "unchanged":
+                    logging.debug("%s %s items:", key.upper(), result_key)
+                    for each in result[result_key]:
+                        logging.debug("    %s", each)
+
+    logging.info("NOTE: Please make sure to fill in the constant files. Feel free to use helper functions to do it")
+    logging.info("NOTE: We also advise to check authorization settings")
+    logging.debug("NOTE: All the paths mentioned use %s as root directory", results_path)
+    # logging.info("==== NOTE: ====")
+    # logging.info("- Please make sure to fill in the constant files. Feel free to use helper functions to do it")
+    # logging.info("- We also advise to check authorization settings")
+    # logging.debug("- All the paths mentioned use %s as root directory", results_path)
+
+
+def log_result_named(results_path):
+    """Function: log run results"""
+
+    result = {
+        "folders": [],
+        "files": [],
+        "classes": [],
+        "functions": [],
+    }
+
+    for root, dirs, files in os.walk(results_path):
+        result["folders"] += [os.path.join(root, x) for x in dirs]
+
+        for filename in files:
```

Comment:
you should check the file extension to be .py

Reply:
Do we generate .py files only?

Reply:
you are right

Reply:
Done: https://github.com/vsvirsky/swagger2locustio/pull/75/commits/fa5914e20786d06ffd867aca348d815c80430f55